src/Module.Server/Rewards/CrpgRewardClient.cs

```diff
@@ -43,6 +43,12 @@ protected override void AddRemoveMessageHandlers(GameNetwork.NetworkMessageHandl
     private void HandleRewardUser(CrpgRewardUser message)
     {
         var reward = message.Reward;
+        if (message.LowPopulation)
+        {
+            InformationManager.DisplayMessage(new InformationMessage("Low Population! Repair Are Free!",
+                new Color(65, 105, 225)));
+        }
+

```

Comment:
What do you think of merging that into the gold reward message. For example
```
Gained 45 gold (reward: 45, upkeep: free, low population server)
```

Reply:
That's a really nice idea

Reply:
it's done